src/css/skins/seven.less

```diff
@@ -1,71 +0,0 @@
-@import "../shared-imports/vars";
-@import "../controls/imports/transitions";
-
-.jw-skin-seven {
-    @display-bkgd-color: @controlbar-background;
-
-    .jw-display-icon-container {
-        border-radius: 3.5em;
```

Comment:
You might consider moving some of these styles to the appropriate class defaults (like styles in display.less). Just make sure you're only adding or replacing (not duplicating) a style like `border-radius`.